lib/foreman/renderer/scope/report.rb

```diff
@@ -2,6 +2,61 @@ module Foreman
   module Renderer
     module Scope
       class Report < Foreman::Renderer::Scope::Base
+        def initialize(**args)
+          super
+          @report_data = []
+          @report_headers = []
+        end
+
+        def report_render(format: :csv)
```

Comment:
also this and report_row needs to be enabled now in Safemode, though for reports templates only